### DISCUSSION TITLE
Unable to build using bun

### DISCUSSION THREAD

**Discussion by Lorenzo0111** [vercel/community#7281] [Bug Report]
### Page to Investigate

/

### Steps to Reproduce

I'm trying to build an app with bun and hono following https://github.com/vercel-community/bun but it gives an error saying I should contact the support

**Comment by pawlean** [comment]
Hi @Lorenzo0111! 

Could you share some more information on what errors you're seeing? It'd help us debug if the template needs updating or if it's a Vercel issue.

Thanks!

[a comment by an automated account is not shown]
